CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 - An option to `UVData.reorder_blts` to sort the autos before the crosses.
 - Added new functionality to `UVData.check` to verify that auto-correlations are real-only,
 along with an option to force them to be real-only if non-zero imaginary components are detected.
+- Added an absolute time tolerance parameter `time_atol` in `UVFlag.to_baseline()`
```

Comment:
This should go at the top of this section (the changelog is in reverse chronological order).

Reply:
Done.